test/ex_doc/retriever/erlang_test.exs

```diff
@@ -93,7 +93,7 @@ defmodule ExDoc.Retriever.ErlangTest do
       assert function1.doc_file =~ "mod.erl"
 
       assert Erlang.autolink_spec(hd(function1.specs), current_kfa: {:function, :function1, 0}) ==
-               "function1() -> <a href=\"https://www.erlang.org/doc/man/erlang.html#type-atom\">atom</a>()."
```

Comment:
These links weren't updated in the previous PR since I missed these EEP 0059 tests. I still can't make them run with my Erlang installation, and it doesn't seem like CI is running them either since CI passed on the prev PR.

Reply:
Some of the tests are failing:

```console
$ mix test test/ex_doc/retriever/erlang_test.exs 
Running ExUnit with seed: 710493, max_cases: 16

.

  1) test docs_from_modules/2 types (ExDoc.Retriever.ErlangTest)
     test/ex_doc/retriever/erlang_test.exs:274
     Assertion with == failed
     code:  assert type1.spec |> Erlang.autolink_spec(current_kfa: {:type, :type1, 0}) ==
              "type1() :: <a href=\"https://www.erlang.org/doc/man/erlang.html#type-atom\">atom</a>()."
     left:  "type1() :: <a href=\"https://www.erlang.org/doc/apps/erts/erlang.html#t:atom/0\">atom</a>()."
     right: "type1() :: <a href=\"https://www.erlang.org/doc/man/erlang.html#type-atom\">atom</a>()."
     stacktrace:
       test/ex_doc/retriever/erlang_test.exs:308: (test)

.

  2) test docs_from_modules/2 function with no docs is generated (ExDoc.Retriever.ErlangTest)
     test/ex_doc/retriever/erlang_test.exs:216
     ** (MatchError) no match of right hand side value: {[], []}
     code: {[mod], []} = Retriever.docs_from_modules([:mod], %ExDoc.Config{})
     stacktrace:
       test/ex_doc/retriever/erlang_test.exs:224: (test)

....

  3) test docs_from_modules/2 module with no docs is generated (ExDoc.Retriever.ErlangTest)
     test/ex_doc/retriever/erlang_test.exs:11
     match (=) failed
     code:  assert {[_], []} = Retriever.docs_from_modules([:mod], %ExDoc.Config{})
     left:  {[_], []}
     right: {[], []}
     stacktrace:
       test/ex_doc/retriever/erlang_test.exs:16: (test)

.

  4) test docs_from_modules/2 records (ExDoc.Retriever.ErlangTest)
     test/ex_doc/retriever/erlang_test.exs:318
     ** (MatchError) no match of right hand side value: {[], []}
     code: {[mod], []} = Retriever.docs_from_modules([:mod], config)
     stacktrace:
       test/ex_doc/retriever/erlang_test.exs:337: (test)

.
Finished in 0.4 seconds (0.4s async, 0.00s sync)
12 tests, 4 failures
$ elixir -v
Erlang/OTP 27 [erts-15.0] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit]

Elixir 1.17.0-rc.0 (a2600ea) (compiled with Erlang/OTP 27)
```

Reply:
They passed for me here but I am running against Erlang main. I will try to reproduce on a 27 release later!